config/dev.exs

```diff
@@ -1 +1,8 @@
 import Config
+
+if System.get_env("NAMESPACE") do
```

Comment:
can't we do this in the mix task?
We can, we can do this in the mix task.

Reply:
The thing is that the `mix release lexical` steps for release always runs *after* the `mix compile` and this needs to be cleaned before `mix compile`, here it does run before compile.

Reply:
Release is totally configurable. You can even Alia's it.